lisa/wlgen/rta.py

```diff
@@ -151,6 +157,14 @@ def by_profile(cls, target, name, profile, res_dir=None, default_policy=None,
           be an integer value or a CPU string (e.g. "CPU0").
         :type calibration: int or str
 
+        :param log_stats: Generate a log file with stats for each task
+        :type log_stats: bool
+
+        :param trace_events: A list of trace events to generate.
+          The full list of trace events generated by rt-app is defined by
+          :class:RTA's `trace_events`. By default, no events are generated.
```

Comment:
```s/:class:RTA/:class:`RTA`/```

If `trace_events` is a class attribute, it should be possible to reference it using ```:attr:`RTA.trace_events` ```

Reply:
Nope, that attribute has been dropped in the last version, lemme fix the comment accordingly.